en/docs/guides/Provisioning/outbound-provisioning.md

```diff
@@ -0,0 +1,10 @@
+# Outbound Provisioning
```

Comment:
I personally feel that since the nav is now more organized, we don't need this overview page here and users can simply go directly to the page they want...WDYT?

Reply:
During discussions for user management, we decided that we need an overview even if the content is less because there might be other topic that should be linked off to a generic overview page. So followed the same here. WDYT?